paasta_tools/kubernetes/bin/cleanup_nodes.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+# Copyright 2015-2019 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Usage: ./cleanup_nodes.py [options]
+
+Command line options:
+
+- -v, --verbose: Verbose output
+- -n, --dry-run: Only report what would have been deleted
+"""
+import argparse
+import logging
+import sys
+from typing import List, Tuple, Union
+from kubernetes.client import V1Node, V1DeleteOptions
+from kubernetes.client.rest import ApiException
+from paasta_tools.kubernetes_tools import KubeClient
+from paasta_tools.kubernetes_tools import get_all_nodes
+from paasta_tools.utils import load_system_paasta_config
+
+log = logging.getLogger(__name__)
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(description="Removes stale kubernetes CRDs.")
+    parser.add_argument(
+        "-v", "--verbose", action="store_true", dest="verbose", default=False
+    )
+    parser.add_argument(
+        "-n", "--dry-run", action="store_true", dest="dry_run", default=False
+    )
+    args = parser.parse_args()
+    return args
+
+
+def nodes_for_cleanup(nodes: List[V1Node]) -> List[V1Node]:
```

Comment:
Do you really need this? It's a one line function and probably won't be used outside of this script.

Reply:
eh, I think it's fine